cmd/miniooni/main.go

```diff
@@ -145,7 +143,15 @@ type logHandler struct {
 }
 
 func (h *logHandler) HandleLog(e *log.Entry) (err error) {
-	s := fmt.Sprintf("<%s> %s", e.Level, e.Message)
+	if e.Level > log.InfoLevel {
+		wrongCounter++
+	}
```

Comment:
This is not the correct way to count the number of failed tests. To this end, you need to count the number of times running a test returns a non `nil` error. There is no guarantee anywhere that a test that fails will emit just a single log event with log severity being equal to `warning`. What's more, a warning is not necessarily a failure. Hence, this way of counting the number of failures is extremely fragile and should not be used.

Reply:
OK! But `warning` and `error` levels are a serious issue that should alert a user who never uses `-v`.

Reply:
> But warning and error levels are a serious issue that should alert a user who never uses -v

As I explained in https://github.com/ooni/probe-engine/pull/35#discussion_r318077908, if we let apex/log do its job, warning and error logs are _always_ shown.